### ISSUE TITLE
About the final test_error

### ISSUE THREAD

**Issue by catwang1207** [katsura-jp/efficientnet-pytorch#13]
I use the net to train cifar-100, but the after 300 epoch the train_error reaches to 0 but the test_error is still about 55%（acually it reach 55% after 50 epoch）. The paper said that the effcientnet for cifar-100 accuracy can reach 88.1%. What's the problem?
I'm using efficientnet-b3 and the optimizer is torch.optim.SGD(model.parameters(),lr=0.1.momentum=0.9,weight_decay=1e-5)
